gradle/interpreters.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+task copyJarDependencies(type: Copy) {
+  description = 'Copies all dependencies to the interpreter directory.'
+  def proj = project.name
+  if (project.name == "markdown") {
+    proj = "md"
+  }
+  if (project.name == "shell") {
+    proj = "sh"
+  }
+  
+  def libDir = new File(rootProject.projectDir, "/interpreter/$proj")
+  configurations.runtime.filter {it.name.endsWith 'jar'}.each { 
+      File file -> moveJarIntoInterpreterDirectory(file, libDir)
+    }
+}
+
+void moveJarIntoInterpreterDirectory(File file, libDir) {
+  copy {
+    from file
+    into libDir
+    /**
+     * Super important for version conflict with netty... 
+     * spend 10 f*cking minutes before noticing this...
+     */
+    exclude '**netty-3.2.2.Final**'
```

Comment:
this little thing is bring by a subsubsubsub dependency and I couldnt find a way to get rid of it.